command/cp.go

```diff
@@ -423,6 +444,9 @@ func (c Copy) Run(ctx context.Context) error {
 		srcurl := object.URL
 		var task parallel.Task
 
+		c.progressbar.AddTotalBytes(object.Size)
```

Comment:
If we upload/download a single file progress bar only shows the total number of objects which is always 1. Is it possible to update `object.Size` so it is > 0.

![Uploading image.png…]()

Reply:
Resolved. https://github.com/peak/s5cmd/pull/590/commits/31959d1fbbe8c788b8d7709fa969afb7bd004637